plugin/Cargo.toml

```diff
@@ -13,7 +13,7 @@ crate-type = [ "dylib" ]
 plugin = true
 
 [dependencies]
-mutagen = "0.1.0"
+mutagen = {"version" = "0.1.0", "path" = ".."}
```

Comment:
I've doubts here: @llogiq you change this line few days ago. How this should work? Should we avoid this and wait to publish another version of the lib? It would make sense to merge lib and plugin library?

Reply:
I think we should bump the version and publish all crates.

I'd also suggest to keep the version numbers of all crates in sync, to keep things simple.